src/components/subscriptions/tests/licenses/LicenseAllocationHeader.test.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/prop-types */
+import { screen, render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import React from 'react';
+import { SubscriptionDetailContext } from '../../SubscriptionDetailContextProvider';
+import { SubsidyRequestConfigurationContext } from '../../../subsidy-request-configuration';
+
+import LicenseAllocationHeader from '../../licenses/LicenseAllocationHeader';
+
+import { SUPPORTED_SUBSIDY_TYPES } from '../../../../data/constants/subsidyRequests';
+
+const BNR_NEW_FEATURE_ALERT_TEXT = 'browse and request new feature alert!';
+jest.mock('../../licenses/NewFeatureAlertBrowseAndRequest', () => ({
+  __esModule: true,
+  default: () => BNR_NEW_FEATURE_ALERT_TEXT,
+}));
+
+const mockSubscription = {
+  licenses: {
+    allocated: 3,
+    total: 5,
+  },
+};
+const mockSubsidyRequestConfiguration = {};
+
+const LicenseAllocationHeaderWrapper = ({
+  subscription = mockSubscription,
+  subsidyRequestConfiguration = mockSubsidyRequestConfiguration,
+}) => (
+  <SubscriptionDetailContext.Provider value={{ subscription }}>
+    <SubsidyRequestConfigurationContext.Provider value={{ subsidyRequestConfiguration }}>
+      <LicenseAllocationHeader />
+    </SubsidyRequestConfigurationContext.Provider>
+  </SubscriptionDetailContext.Provider>
+);
+
+describe('<LicenseAllocationHeader />', () => {
+  it('should render license allocation', () => {
+    render(<LicenseAllocationHeaderWrapper />);
+    expect(screen.getByText('3 of 5 licenses allocated'));
+  });
+
+  it.each([
```

Comment:
clarification: What about the case where no `subsidyRequestConfiguration` exists yet?